src/Features/Core/Portable/SolutionCrawler/IDocumentDifferenceService.cs

```diff
@@ -8,13 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.SolutionCrawler;
 
-internal class DocumentDifferenceResult(InvocationReasons changeType, SyntaxNode? changedMember = null)
-{
-    public InvocationReasons ChangeType { get; } = changeType;
-    public SyntaxNode? ChangedMember { get; } = changedMember;
-}
-
 internal interface IDocumentDifferenceService : ILanguageService
 {
-    Task<DocumentDifferenceResult?> GetDifferenceAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken);
+    Task<SyntaxNode?> GetChangedMemberAsync(Document oldDocument, Document newDocument, CancellationToken cancellationToken);
 }
```

Comment:
now begins a long series of solution-crawler files that are just fully deleted :)